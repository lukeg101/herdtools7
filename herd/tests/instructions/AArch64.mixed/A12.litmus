AArch64 A12
(* Store immediate, with offset, symbolic location *)

{ int64_t x; 0:x0=1; 0:x2=x;}

P0;
  STR W0, [X2, #4];

exists (x=4294967296)
