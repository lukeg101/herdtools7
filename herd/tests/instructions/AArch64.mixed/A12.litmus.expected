Test A12 Allowed
States 1
x=0x100000000;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (x=0x100000000)
Observation A12 Always 1 0
Hash=86545e2af1a0880e99658d8d4d00c83c

